zerver/data_import/slack.py

```diff
@@ -1262,12 +1270,37 @@ def get_timestamp_from_message(message: ZerverFieldsT) -> float:
     return float(message["ts"])
 
 
+def is_integration_bot_message(message: ZerverFieldsT) -> bool:
+    return message.get("subtype") == "bot_message" and "user" not in message and "bot_id" in message
+
+
+def convert_bot_info_to_slack_user(bot_info: dict[Any, Any]) -> ZerverFieldsT:
+    bot_user = {
+        "id": bot_info["id"],
+        "name": bot_info["name"],
+        "deleted": bot_info["deleted"],
+        "is_mirror_dummy": False,
+        "real_name": bot_info["name"],
+        "is_integration_bot": True,
+        "profile": {
+            "image_72": bot_info["icons"]["image_72"],
```

Comment:
What resolution will this image have? Is there a reason it's called `image_72`? Is it 72x72? In general, we want a higher-resolution image than what we show in the message feed, at least 500x500, since that's the maximum that Zulip shows in places like the user profile for a user.

Reply:
(If the answer is this is the best available, we should note that in a comment... but also maybe look at whether there's an API to ask for a higher-resolution one)

Reply:
hmm, it seems like it stands for `72x72` pixels as the preview in their user profile is quite blurry. (added some screenshots in the PR description)

And, this is likely the highest resolution we can get for the bots. We can only call [this](https://api.slack.com/methods/bots.info) endpoint for more info on them.

Reply:
Added code comment for this:

```python
def convert_bot_info_to_slack_user(bot_info: dict[Any, Any]) -> ZerverFieldsT:
    # We use "image_72," an icon-sized 72x72 pixel image, for this bot user's
    # avatar because it is the best available option. This will make the
    # avatar appear blurry in places where a medium-sized avatar (500x500px)
    # is expected, such as in the user profile menu.
```

Reply:
OK, nothing to be done but document it, thanks.